packages/gatsby-theme-api-docs/src/components/type/children.js

```diff
@@ -22,30 +28,14 @@ const Children = ({
           />
         ) : null}
 
-        {apiType.enumeration || apiType.description ? (
-          <Enum
-            description={apiType.description}
-            values={apiType.enumeration}
-          />
-        ) : null}
-      </SpacingsStack>
-    );
-  }
-
-  return (
-    <SpacingsStack scale="m">
-      {apiType.enumeration || apiType.description ? (
-        <Enum description={apiType.description} values={apiType.enumeration} />
-      ) : null}
+        {apiType.examples ? <Examples examples={apiType.examples} /> : null}
+      </SideBySide>
 
-      {apiType.properties ? (
-        <Properties
-          apiType={apiType}
-          parentDiscriminator={parentDiscriminator}
-        />
+      {apiType.description && renderDescriptionBelowProperties ? (
+        <Description>{apiType.description}</Description>
       ) : null}
 
-      {apiType.examples ? <Examples examples={apiType.examples} /> : null}
+      {apiType.enumeration ? <Enum values={apiType.enumeration} /> : null}
```

Comment:
Previously, descriptions and enums were either rendered on top or below properties, so does this change mean we want descriptions to always be rendered last?

Reply:
This was more an accident I think .  I suggest to leave it unchanged here and you do it proper in your Enums PR